### PR TITLE
[MRESOLVER-92] Revert MRESOLVER-7

### PR DESCRIPTION
**Pull request by Tibor17**
Jenkins build
https://builds.apache.org/job/maven-box/job/maven-resolver/job/pre-MRESOLVER-92/

### CODE REVIEW

File: maven-resolver-impl/src/main/java/org/eclipse/aether/internal/impl/collect/DefaultDependencyCollector.java [hunk 1]
```diff
@@ -86,12 +79,13 @@
     implements DependencyCollector, Service
 {
 
-    static final String CONFIG_PROP_MAX_EXCEPTIONS = "aether.dependencyCollector.maxExceptions";
+    private static final String CONFIG_PROP_MAX_EXCEPTIONS = "aether.dependencyCollector.maxExceptions";
+
+    private static final int CONFIG_PROP_MAX_EXCEPTIONS_DEFAULT = 50;
```
Comment (michael-o) [inline maven-resolver-impl/src/main/java/org/eclipse/aether/internal/impl/collect/DefaultDependencyCollector.java]: Where does this one come from?

File: maven-resolver-impl/src/main/java/org/eclipse/aether/internal/impl/collect/DefaultDependencyCollector.java [hunk 2]
```diff
@@ -86,12 +79,13 @@
...
 
-    static final String CONFIG_PROP_MAX_CYCLES = "aether.dependencyCollector.maxCycles";
+    private static final String CONFIG_PROP_MAX_CYCLES = "aether.dependencyCollector.maxCycles";
 
-    private static final String CONFIG_PROP_THREADS = "aether.artifactDescriptor.threads";
-    private static final int DEFAULT_THREADS = 5;
+    private static final int CONFIG_PROP_MAX_CYCLES_DEFAULT = 10;
```
Comment (michael-o) [inline maven-resolver-impl/src/main/java/org/eclipse/aether/internal/impl/collect/DefaultDependencyCollector.java]: Where does this one come from?

File: maven-resolver-impl/src/main/java/org/eclipse/aether/internal/impl/collect/DefaultDependencyCollector.java (hunk 1 above)
Comment (Tibor17) [reply maven-resolver-impl/src/main/java/org/eclipse/aether/internal/impl/collect/DefaultDependencyCollector.java]: It's from checkstyle failure. When I was building the project the `maven-checkstyle-plugin` was wanted be to declare a constant for unknown number and string.
No idea how this project could be built without it till now.

File: maven-resolver-impl/src/main/java/org/eclipse/aether/internal/impl/collect/DefaultDependencyCollector.java (hunk 2 above)
Comment (Tibor17) [reply maven-resolver-impl/src/main/java/org/eclipse/aether/internal/impl/collect/DefaultDependencyCollector.java]: The same as before.

**Comment by jira-importer** [comment]
Resolve #798

**Pull request merged**